cpu/esp_common/esp-wifi/esp_wifi_netdev.c

```diff
@@ -818,34 +906,45 @@ void esp_wifi_setup (esp_wifi_netdev_t* dev)
      */
     /* activate the Station and the SoftAP interface */
     result = esp_wifi_set_mode(WIFI_MODE_APSTA);
-#else /* MCU_ESP8266 */
+#else /* defined(MCU_ESP8266) */
     /* activate only the Station interface */
     result = esp_wifi_set_mode(WIFI_MODE_STA);
-#endif /* MCU_ESP8266 */
+#endif /* defined(MCU_ESP8266) */
     if (result != ESP_OK) {
         ESP_WIFI_LOG_ERROR("esp_wifi_set_mode failed with return value %d", result);
         return;
     }
 
-#ifdef MCU_ESP8266
+#if defined(MCU_ESP8266) || defined(MODULE_ESP_WIFI_AP)
+#ifdef ESP_WIFI_AP_PREFIX
+    uint8_t mac[ETHERNET_ADDR_LEN];
+    esp_wifi_get_mac(ESP_MAC_WIFI_SOFTAP, mac);
+    sprintf((char*)wifi_config_ap.ap.ssid, "%s%02x%02x%02x%02x%02x%02x",
+            ESP_WIFI_AP_PREFIX,
+            mac[0], mac[1], mac[2],
+            mac[3], mac[4], mac[5]);
```

Comment:
Why do you keep it soooo small? 
```suggestion
            ESP_WIFI_AP_PREFIX, mac[0], mac[1], mac[2], mac[3], mac[4], mac[5]);
```

Reply:
Fear of the 80-chars limit :wink: